build.gradle

```diff
@@ -279,6 +279,8 @@ model {
                     cppCompiler.args "/Zc:throwingNew"          // Assume new throws on error
                     cppCompiler.args "/W3"                      // Enable lots of warnings, disbale individual warnings with /WD<NUM>
                     cppCompiler.args "/WX"                      // Warnings are errors
+                    cppCompiler.args "/D_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING"
```

Comment:
Is that necessary or did the problem only occur on Windows 24?

Reply:
This is a warning with C++17, so it's not related to the problem we've seen with the `windows24` machine.